cmd/operator_install.go

```diff
@@ -33,8 +33,12 @@ func newOperatorInstallCmd(operatorConfig *operatorCommandConfig) *cobra.Command
 	// installCmd represents the "appsody deploy install" command
 	var installCmd = &cobra.Command{
 		Use:   "install",
-		Short: "Install the Appsody Operator into the configured Kubernetes cluster",
-		Long:  ``,
+		Short: "Install the Appsody Operator.",
+		Long: `Install the Appsody Operator into your configured Kubernetes cluster.
+		
+The Appsody Operator listens for incoming AppsodyApplication resources on your cluster. For more information, see https://operatorhub.io/operator/appsody-operator. By default, the operator only watches one namespace, unless the '--watch-all' flag is used. Additional operators must be installed to watch additional namespaces.`,
+		Example: `  appsody operator install --namespace my-namespace --watchspace my-watchspace
```

Comment:
"By default, the operator only watches one namespace, unless the '--watch-all' flag is used. Additional operators must be installed to watch additional namespaces."

I found these sentences a bit puzzling.

Can we separate these out into 1. default behaviour, 2. what happens with the --watch-all flag 3. how/when you get into a situation where you need to specify another operator to watch another namespace?

"By default, the operator only watches one namespace. If the '--watch-all' flag is specified, the operator watches all namespaces."

If I'd specified the `--watch-all` flag surely I wouldn't need to add another namespace? So is 3. only required if you're watching one namespace and want to watch another namespace (but didn't specify --watch-all)?

Reply:
Case 3 is if you want to watch more than one namespace, but not all namespaces. I have made the changes to reflect this